### ISSUE TITLE
Unable to use TestProject Addons with C# OpenSDK

### ISSUE THREAD

**Issue by kishangupta-15** [testproject-io/csharp-opensdk#243]
There is no way to use TestProject Platform's Addons with C# OpenSDK

**Comment by mstrelex** [comment]
What are you trying to do? Run an addon using a proxy from within a C# coded test?

**Comment by kishangupta-15** [comment]
Yes, I want to use PDF Actions addon for now in my C# coded test.
There are few other addons too which I am looking forward to use though like Random Data Generator, Database addon etc. but right now I need to use PDF Actions for PDF validations in my coded C# test.

**Comment by mstrelex** [comment]
Right now it is not possible to generate an addon proxy for C# OpenSDK.
It is also in our enhancements list and we will inform you when it will be implemented.

**Comment by kishangupta-15** [comment]
Okay. So after this enhancement is implemented, will I able able to use existing TestProject addons (which are created in Java using Legacy SDK) or do I have to write new addons in C# and then use them?

Also, currently is this functionality available in Java OpenSDK?

**Comment by mstrelex** [comment]
Yes, the way addon proxies are designed allows you to use them from any other Open SDK.

Python - https://github.com/testproject-io/python-opensdk#addon-proxy
Java - https://github.com/testproject-io/java-opensdk#addon-proxy

**Comment by kishangupta-15** [comment]
Okay, thank you. Looking forward for this enhancement to be implemented to use existing proxies.
